gef-ui/server/controllers/unissued-facilities/validation.js

```diff
@@ -57,7 +58,11 @@ const facilityValidation = async (body, query, params, facility) => {
   if (application.manualInclusionNoticeSubmissionDate) {
     const minSubmissionDate = (new Date(Number(application.manualInclusionNoticeSubmissionDate))).setHours(0, 0, 0, 0);
```

Comment:
Please add a comment for `MIN` to make it uniform.
Just a single line comment please.

```
\\ If MIN, then MIN submission date plus three months

Reply:
done